phoenix/block-templates/404.html

```diff
@@ -0,0 +1,23 @@
+<!-- wp:template-part {"slug":"header","theme":"phoenix","tagName":"header"} /-->
```

Comment:
To satisfy [recent changes in Gutenberg](https://github.com/WordPress/gutenberg/pull/28088) (as of 9.9) the "theme" attribute should be removed from the template-part blocks.